gpiozero/output_devices.py

```diff
@@ -300,7 +299,10 @@ def __init__(self, pin=None, active_high=True, initial_value=0, frequency=100):
         self._controller = None
         if not 0 <= initial_value <= 1:
             raise OutputDeviceBadValue("initial_value must be between 0 and 1")
-        super(PWMOutputDevice, self).__init__(pin, active_high)
+        if initial_value in (0, 1):
+            super(PWMOutputDevice, self).__init__(pin, active_high, initial_value=initial_value)
```

Comment:
@lurch precisely - and that's also why I haven't added one yet - I vaguely recall trying it a while back and realizing it wasn't a 30 second job so I left it for the time being (on the assumption that, given we only had software PWM at the time anyway blips were par for the course anyway :)